### DISCUSSION TITLE
Use of website content by Google after running lighthouse

### DISCUSSION THREAD

**Discussion by adamstone7** [GoogleChrome/lighthouse#11859] [Q&A] [answered]
Do Google Lighthouse or other Google services store information related to website content or metrics following the use of Lighthouse?

**Comment by paulirish** [comment, marked as answer]
I think you're asking if Google is collecting/keeping data about a website after it's been analyzed by Lighthouse?

The short answer is nope.  But it's worth accounting for the different "clients" of Lighthouse.

If you're using the Lighthouse CLI or the Lighthouse node module, then everything is local and Google has no idea the software is even being used*. …The one exception to that is if you [opt-in to error reporting](https://github.com/GoogleChrome/lighthouse/blob/master/docs/error-reporting.md). (If you didn't opt in, nothing's reported). But even with error reporting, it's only some machine and error metadata that's reported.. The URL that was analyzed **is** reported, but nothing like the content of the webpage.

If you're using Lighthouse in Chrome DevTools (or Edge DevTools), everything is local. And there's no error reporting there.

[PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) (PSI) is a service by Google powered by Lighthouse (and run by an adjacent team). When you use PSI, Google servers fetching the page and run Lighthouse against it. It's fair to say that PSI stores some aspects of the webpage content afterwards. If you look at the JSON payload of the PSI API, then that's the data that the PSI service has and keeps around, at least for logging and analysis purposes.  Also FWIW, there's no connection (that I know of) between PSI and the Google Search index.

If I had a website I didn't want Google services to know about at all, then I wouldn't use PageSpeed Insights and I'd stick to the Lighthouse CLI (or the Lighthouse panel in DevTools). :) Hope that helps!

**Comment by adamstone7** [comment]
Hi Paul,

Thank you very much for your detailed and prompt response. This answered my question perfectly, and I'm quite encouraged  by the answer! Quite often the Indigenous communities I work with have sensitive cultural knowledge they want to be able to distribute online amongst themselves, and within reason, don't want to risk having things happen to their information that they aren't aware of.

All the best,

Adam

Adam Stone, M.Cog.Sc., PhD
Researcher, Atlas of the Languages of Iran
Geomatics and Cartographic Research Centre, Carleton University
